Ability to customise the title/main heading

jamesmacwhite opened the pull request:
Currently the `<title>` and `<h1>` value is hard coded, this allows the user to change it to whatever they desire. I've used a small function with `isset()` to essentially check if `title_text` is set and if it is, use it as value for these elements. A default title value is set in functions.php for older configurations which will not have this config value to break a blank value.

jamesmacwhite committed:
Ability to customise the title/main heading

craigwatson commented on php/functions.php:
```diff
@@ -471,3 +471,20 @@ function getProcessUptime($process)
+ *
+ * @return string
+ */
+function getTitleText()
```
This function is relatively superfluous, the template should echo the config value directly, to keep in-style with the rest of the code.

Admittedly, echoing values in-line into HTML isn't the _best_ way to do this, but in the interests of simplicity and lightness it's nicer than relying on Twig.

jamesmacwhite commented:
@craigwatson I have updated this PR. I have made it more like the rest of the template. However I have used the [Ternary operator](https://davidwalsh.name/php-shorthand-if-else-ternary-operators) for short hand if/else on the title parts, because it seems overkill to use full if else for a single line of text.

I've also fixed the merge conflict with `README.md`. Let me know if this is OK.
